src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationService.java

```diff
@@ -73,4 +75,15 @@ public MigrationComplaintCorrespondent getPrimaryCorrespondent(LinkedHashMap cor
         return primaryCorrespondent;
     }
 
+    public List<MigrationComplaintCorrespondent> getAdditionalCorrespondents(Optional<String> correspondentJson)  {
```

Comment:
Is there no additional logic required here - will all the data within a correspondent be required? I'm wondering if a DTO would be useful rather than `string`?